docs/core/tools/dotnet-workload-install.md

```diff
@@ -0,0 +1,104 @@
+---
+title: dotnet workload install command
+description: The 'dotnet workload install' command installs optional workloads.
+ms.date: 07/08/2021
+---
+# dotnet workload install
+
+**This article applies to:** ✔️ .NET 6 Preview SDK and later versions
+
+## Name
+
+`dotnet workload install` - Installs optional workloads.
+
+## Synopsis
+
+```dotnetcli
+dotnet workload install <WORKLOAD_ID>
+    [--add-source <SOURCE>] [--configfile <FILE>] [--disable-parallel]
+    [--download-to-cache <CACHE>] [--from-cache <CACHE>]
+    [--ignore-failed-sources] [--include-previews] [--interactive]
+    [--no-cache] [--sdk-version <VERSION>] [--skip-manifest-update]
+    [--temp-dir <PATH>] [-v|--verbosity <LEVEL>]
+
+dotnet workload install -?|-h|--help
+```
+
+## Description
+
+The `dotnet workload install` command installs an *optional workload*. Optional workloads cn be installed on top of the .NET SDK to provide support for various application types, such as [.NET MAUI](/dotnet/maui/what-is-maui) and [Blazor WebAssembly AOT](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#blazor-webassembly-ahead-of-time-aot-compilation).
+
+Use [dotnet workload search](dotnet-workload-search.md) to learn what workloads are available to install.
+
+The `dotnet workload install` command copies the workloads from NuGet.org into your SDK install folder, so it needs to run elevated if that folder requires administrator write permissions.
```

Comment:
> The `dotnet workload install` command copies the workloads from NuGet.org into your SDK install folder

This is not quite right. It would be more accurate to say that the command installs workload packs, which are packaged and hosted on Nuget.org.

> it needs to run elevated if that folder requires administrator write permissions.

This is true for mac and linux SDK installations which are installed to a protected directory, but depending on the location of the install a user may not need to be elevated. Also, on windows a user will not have to elevate even if the SDK is installed to program files, as we have a special installer which will use MSIs to install in that case.